deb/ch3/ep1/answers/passengers.py

```diff
@@ -0,0 +1,46 @@
+from deb.utils.config import config
+from deb.utils.logging import logger
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import initcap, col, sha2, concat_ws
+
+# ===============================================================
+# ‹‹=‹‹=‹‹=‹‹=‹‹=‹‹=‹‹=‹‹= RUN FUNCTIONS =››=››=››=››=››=››=››=››
+# ===============================================================
+
+def run():
+    # Build session
+    sparkql = SparkSession.builder.master('local').getOrCreate()
+
+    # Load config informaiton if __name__ == '__main__':
+    people_path = config['defaults']['ch3']['ep1']['passenger_input'].get(str)
+    save_path = config['defaults']['ch3']['ep1']['passenger_output'].get(str)
+    logger.info(f"Loading passenger info from {people_path}")
+    passengers_df = sparkql.read.csv(people_path, header=True)
+    logger.info(f"There are {passengers_df.count()} rows")
+
+    # Load the passenger data and make sure the names have initial capitalization
+    logger.info("Cleaning names and creating full name")
+    passengers_df = passengers_df.withColumn('first_name', initcap(col('first_name')))\
```

Comment:
Please make sure you explain how `.withColumn()` transforms works in your description.

Reply:
I went into some detail in the intro notebook; when I add the intro section to the site, that should cover it. In my write up for this section I can reference the intro where it's covered.